guides/common/modules/proc_configuring-your-deployment-to-run-ansible-roles.adoc

```diff
@@ -35,8 +35,8 @@ If you want to use custom or third party Ansible roles, ensure to configure an e
 --enable-foreman-proxy-plugin-ansible
 ----
 
-. Distribute SSH keys to enable {SmartProxies} to connect to hosts using SSH.
-For more information, see {ManagingHostsDocURL}ssh-keys-for-remote-execution-overview_managing-hosts[Distributing SSH Keys for Remote Execution] in the _Managing Hosts_ guide.
+. Distribute Script keys to enable {SmartProxies} to connect to hosts using Script.
+For more information, see {ManagingHostsDocURL}ssh-keys-for-remote-execution-overview_managing-hosts[Distributing Script Keys for Remote Execution] in the _Managing Hosts_ guide.
```

Comment:
```suggestion
For more information, see {ManagingHostsDocURL}ssh-keys-for-remote-execution-overview_managing-hosts[Distributing Script Keys for Remote Execution] in _{ManagingHostsDocTitle}_.
```

I assume SSH keys remain SSH keys, so you probably have to change this as well. If you do, you can simply checkout the file from the master branch and ignore my suggestion.